apollo-portal/src/main/resources/static/i18n/en.json

```diff
@@ -41,8 +41,9 @@
   "Common.Operation": "Operation",
   "Common.Delete": "Delete",
   "Component.DeleteNamespace.Title": "Delete Namespace",
-  "Component.DeleteNamespace.PublicContent": "Deleting namespace will cause the instances unable to get the configuration of this namespace. Are you sure to delete it?",
-  "Component.DeleteNamespace.PrivateContent": "Deleting a private Namespace will cause the instances unable to get the configuration of this namespace, and the page will prompt 'missing namespace' (unless the AppNamespace is deleted by admin tool). Are you sure to delete it?",
+  "Component.DeleteNamespace.PublicContent": "Note, that the public Namespace for all environments will be deleted! This will cause the instance and the associated Namespace to fail to obtain the configuration of this Namespace. Are you sure you want to delete it?",
+  "Component.DeleteNamespace.PrivateContent": "Note, that the private Namespace for all environments will be deleted! This will cause the instance to not get the configuration of this namespace. Are you sure you want to delete it?",
+  "Component.DeleteNamespace.LinkedContent": "Note, that the namespace associated with the current environment will be deleted! This will cause the instance to not get the configuration of this namespace. Are you sure you want to delete it?",
```

Comment:
```suggestion
  "Component.DeleteNamespace.PublicContent": "Caution, the public namespace for all environments will be deleted! This will cause the instances unable to get the configuration of this namespace. Are you sure you want to delete it?",
  "Component.DeleteNamespace.PrivateContent": "Caution, the private namespace for all environments will be deleted! This will cause the instances unable to get the configuration of this namespace. Are you sure you want to delete it?",
  "Component.DeleteNamespace.LinkedContent": "Caution, all the namespaces associated with the current environment will be deleted! This will cause the instances unable to get the configuration of this namespace. Are you sure you want to delete it?",
```